pkg/cloudkitty/storageinit.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cloudkitty
+
+import (
+	"github.com/openstack-k8s-operators/lib-common/modules/common/env"
+	telemetryv1 "github.com/openstack-k8s-operators/telemetry-operator/api/v1beta1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// storageInitCommand -
+	// TODO: Once we work on update/upgrades revisit the command in the
+	//       the cloudkitty-storageinit-config.json file.
+	//       If we stop all services during the update/upgrade then we can keep
+	//       the --bump-versions flag.
+	//       If we are doing rolling upgrades we'll need to use the flag
+	//       conditionally (only for adoption) and do the restart cycle of
+	//       services as described in the upstream rolling upgrades process.
+	storageInitCommand = "/usr/local/bin/kolla_set_configs && /usr/local/bin/kolla_start"
+)
+
+// StorageInitJob func
+func StorageInitJob(instance *telemetryv1.CloudKitty, labels map[string]string, annotations map[string]string) *batchv1.Job {
+	args := []string{"-c", storageInitCommand}
+
+	// create Volume and VolumeMounts
+	volumes := GetVolumes("cloudkitty")
```

Comment:
```suggestion
	volumes := GetVolumes(instance.Name)
```
I'll fix that in my PR